Podfile

```diff
@@ -169,7 +169,7 @@ abstract_target 'Apps' do
     ## Gutenberg (React Native)
     ## =====================
     ##
-    gutenberg :tag => 'v1.72.0-alpha5'
+    gutenberg :commit => '2095f19d1a7ed6d52b0f394f9579e66bb4876f0c'
```

Comment:
Should we update to point to https://github.com/wordpress-mobile/gutenberg-mobile/commit/8b29453cdbd5547bb666ab83bf60530f3024ce60 that includes the bundle update among other things? 🤔

Reply:
`2095f19d1a7` should be fine since it is ahead of `8b29453cdb`, That is gutenberg-mobile @`2095f19d1a7` includes all the changes in  `8b29453cdb`

Reply:
Just for completeness, `2095f19d1a7` is not the current head of the Gutenberg Mobile release PR. But `2095f19d1a7` just behind some release note changes so there are no behavioral changes.l

Reply:
Oh, you mean that the bundle change in https://github.com/wordpress-mobile/gutenberg-mobile/pull/4668/commits/6b030ad4fac4a9053dc854307852323e38b7a2e0 is not really a functional change, just changelog changes on the Gutenberg side?

Reply:
Nope. `8b29453` , `6b030ad`, and `2095f19d1` all are from the release script, also in that specific order. `2095f19d1`, being the last commit by the release script, includes all the changes from `6b030ad` and  `8b29453`. 

My last comment was just noting that `2095f19d1` is no longer the head of the GB mobile release branch. But all the later commits are just change log updates. Some are coming from Gutenberg so there are some bundle diffs but again, anything after `2095f19d1` is non behavioral

Reply:
Gotcha, thanks for elaborating Jason! 🙇‍♂️